source/guide_directus.rst

```diff
@@ -126,6 +126,29 @@ Create ``~/etc/services.d/directus.ini`` with the following content:
 
 If it's not in state RUNNING, check your configuration.
 
+Change listening URL
+------------
+
+In ``~/directus/.env`` change the PUBLIC_URL to ``http://0.0.0.0``.
+
+::
+
+  PUBLIC_URL="http://0.0.0.0"
+
+Create web backend
```

Comment:
Please use the snippet: https://github.com/Uberspace/lab/blob/main/STYLE.md#rules-for-guides, e.g. https://github.com/Uberspace/lab/blob/main/source/guide_ghost.rst#configure-web-server